.github/workflows/tox.yml

```diff
@@ -0,0 +1,43 @@
+name: gh
+
+on:
+  create:  # is used for publishing to PyPI and TestPyPI
```

Comment:
I don't see any deployment being done here, are those comments meant for later or am I missing something?

Reply:
I think the idea is for this to run along side Travis for a while, so we'll let Travis to the deploy until we switch over. @nicoddemus 

Correct me if I'm wrong @ssbarnea

Reply:
Yes. That is the plan. In fact code for deploying using github actions is missing from current change. We can switch to github publishing only when we are sure we want to drop travis.

Reply:
OK, thanks folks.